File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcNamedUnit/IfcConversionBasedUnit/IfcConversionBasedUnit.vb
Public Class IfcConversionBasedUnit Inherits IfcNamedUnit
    Public Name As IfcLabel
    Public ConversionFactor As IfcMeasureWithUnit

    ' === EXTENDED BY ===
    ' IfcConversionBasedUnitWithOffset

    ' === INVERSE CLAUSES ===
    ' HasExternalReference : SET [0:?] OF IfcExternalReferenceRelationship FOR RelatedResourceObjects
End Class

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcNamedUnit/IfcConversionBasedUnit/IfcConversionBasedUnit.cs
public class IfcConversionBasedUnit : IfcNamedUnit
{
    public IfcLabel Name { get; set; }
    public IfcMeasureWithUnit ConversionFactor { get; set; }

    // === EXTENDED BY ===
    // IfcConversionBasedUnitWithOffset

    // === INVERSE CLAUSES ===
    // HasExternalReference : SET [0:?] OF IfcExternalReferenceRelationship FOR RelatedResourceObjects
}
